src/libs/actions/MapboxToken.js

```diff
@@ -42,7 +42,7 @@ const setExpirationTimer = () => {
     }, REFRESH_INTERVAL);
 };
 
-const hasTokenExpired = () => moment().isAfter(currentToken.expiration);
+const hasTokenExpired = () => isAfter(new Date(), currentToken.expiration);
```

Comment:
"I think this is incorrect. This should be isAfter(currentToken.expiration, new Date()) instead."
Actually no, this is correct:
We use the isAfter function to compare the current date (new Date()) with currentToken.expiration.
The isTokenExpired variable will be true if the current date is after the expiration date, and false otherwise.

Reply:
sure, one moment

Reply:
<img width="270" alt="image" src="https://github.com/Expensify/App/assets/39777589/bfffb2d5-5e23-4cdb-b5b1-93a0ead22cf8">
@allroundexperts

Reply:
new Date()
new Date(currentToken.expiration)
@allroundexperts

Reply:
Get it, will show now

Reply:
<img width="541" alt="image" src="https://github.com/Expensify/App/assets/39777589/d8d2124e-16a8-44ef-bc66-2f38ca2db1a2">

Reply:
looks correct